packages/main/src/plugin/kubernetes-client.spec.ts

```diff
@@ -1221,19 +1221,23 @@ test('Expect apply with invalid file should error', async () => {
   try {
     await client.applyResourcesFromFile('default', 'missing-file.yaml');
     throw Error('should never get here');
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } catch (err: any) {
+  } catch (err: unknown) {
```

Comment:
This test does not fail if apply does not throw an error, but test name says it 'should'.